### DISCUSSION TITLE
CraftCMS upgrade issue from 3 to 4

### DISCUSSION THREAD

**Discussion by allion-gihan** [craftcms/cms#14369] [Q&A]
Hello Friends,

I'm trying to upgrade CraftCMS 3 to 4 version successfully completed until database migration step, 
When i try to run `php craft migrate/all` command getting the bellow error, 

![image](https://github.com/craftcms/cms/assets/58167686/39736c13-419c-4f92-bb84-38edf74e6430)

columnSuffix column is currently missing in the fields table (before the migration run)

I really appreciate if i get  a generous help from the support and community team to solve this issue.

Regards,
Gihan D. Wijerathna

**Comment by brandonkelly** [comment]
Hi @allion-gihan, would you be able to send your Composer files and a database backup over to support@craftcms.com? We could help look into it from there.

**Reply by allion-gihan** [reply]
@brandonkelly Please find the attachment for the composer file, I sent a email as well. 
[composer.json](https://github.com/craftcms/cms/files/14274742/composer.json)
